### PR TITLE
[ADD] HR Employee Benefits

### PR DESCRIPTION
**Pull request by ghost**
This module is a complete implementation of employee benefits. For now, three types of benefits are managed: fixed amounts, amounts based on a percentage of the gross salary and amounts for each worked hour. See readme file for details.

ping @max3903

### CODE REVIEW

**Comment by ghost** [comment]
The unit tests are passing. Pylint and Flake are fixed in ~~https://github.com/OCA/hr/pull/77.~~

**Comment by ghost** [comment]
~~https://github.com/OCA/hr/pull/83 needs to pass first.~~

**Comment by ghost** [comment]
~~https://github.com/OCA/hr/pull/104 needs to pass first~~

File: hr_employee_benefit/hr_payslip.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+        # Compute the amounts for each employee benefit
+        self.pool['hr.employee.benefit'].compute_amounts(
+            cr, uid, benefit_ids, payslip, context=context)
+
+    def search_benefits(self, cr, uid, payslip, rule, context=None):
+        """ Get all benefits for a given payslip and a given salary rule
+        :type payslip: hr.payslip browse record
+        :type rule: hr.salary.rule browse record
+        :rtype: list of
```
Comment (bwrsandman) [inline hr_employee_benefit/hr_payslip.py]: Trailing white space makes this non-pep8.

Real question, though, is: what is this a list of?

**Comment by mdietrichc2c** [comment]
LGTM :+1:

**Comment by charbeljc** [comment]
LGTM :+1:

**Pull request merged**